compile.sh

```diff
@@ -1,12 +1,18 @@
 ################################################################################
 # This script can be used to compile the main.c when inside TISE directory
 # It creates an executable main.o that needs to be called with the parameter
-# for the TISE simulation
-# like this:
-# ./main.o [int NUM] [double mass] [int potential number] [double tol] [double res]
+# It also compiles all test files and creates their executables.
+# The test executables are created inside the test directory.
 ################################################################################
 
 IDIR="./include"
 MDIR="./modules"
+TDIR="./test"
+SDIR="./scripts"
 
-gcc -o main.o -I ${IDIR} ${MDIR}/geometry.c ${MDIR}/hamiltonian.c ${MDIR}/powermethod.c ${MDIR}/observables.c ${MDIR}/linearalgebra.c ${MDIR}/conjugategradient.c main.c -lm
+gcc -o ${SDIR}/main.o -I ${IDIR} ${MDIR}/geometry.c ${MDIR}/hamiltonian.c ${MDIR}/powermethod.c ${MDIR}/observables.c ${MDIR}/linearalgebra.c ${MDIR}/conjugategradient.c main.c -lm
```

Comment:
I did get an ERROR compiling. But I think it has to do with the missing wavefunction.c